Source/Serilog.Exceptions.Refit/Serilog.Exceptions.Refit.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup Label="Build">
+    <TargetFrameworks>net5.0;netstandard2.0</TargetFrameworks>
+  </PropertyGroup>
+
+  <PropertyGroup Label="Package">
+    <Product>Serilog Exceptions Refit</Product>
+    <Description>Log exception details and custom properties that are not output in Exception.ToString(). Contains custom destructurers for Refit exceptions.</Description>
+    <PackageTags>Serilog;Exception;Log;Logging;Detail;Details;Refit</PackageTags>
+  </PropertyGroup>
+ 
+  <ItemGroup Label="Project References">
+    <ProjectReference Include="..\..\Source\Serilog.Exceptions\Serilog.Exceptions.csproj" />
+  </ItemGroup>
+
+  <ItemGroup Label="Package References">
+    <PackageReference Include="Refit" Version="6.0.94" />
```

Comment:
That's subtle but I think that libraries should not force versions that are not absolutely necessary. If we can change to 6.0.0 here I would consider it more precise.

Reply:
Version 6.0.0 doesn't exist. There is 6.0.1 but there are so few NuGet downloads I'd rather keep the latest version if ok with you ?

Reply:
In my mind, there is absolutely no reason to put anything other than the lowest possible version, `6.0.1` in that case.
Anything else just forces some clients to update.
That said, if you think otherwise, go ahead, that's not a blocking issue.

Reply:
Good point. I downgraded to version 6.0.15. Below this version there is a bug which auto generates refit classes even if there are no defined refit interfaces. This was then causing the static code analysis to block the build on the auto generated classes. Hope this compromise is ok for you.